### PR TITLE
Change colour of added and removed text

### PR DESCRIPTION
**Pull request by samuel-w**
This is what the colours for added files are. Really bright and hard to read.
example
This is what they look like now. Much easier to read.
example2

### CODE REVIEW

**Comment by m3nu** [comment]
How about dark themes?

For links I changed from dark blue to a lighter blue because it works on light and dark equally.

**Comment by samuel-w** [comment]
This is what it looks like in dark mode. Looks fine to me.
example3

**Comment by m3nu** [comment]
Dark red also OK?

**Comment by samuel-w** [comment]
I changed it so that it only sets the colours to dark when running in light mode.

**Review by m3nu — APPROVED** [review]

**Review by m3nu — CHANGES_REQUESTED** [review]
See comment to reduce calling `uses_dark_mode()` repeatedly. May be a heavy function and it would be called *very* often for each line.

File: src/vorta/views/diff_result.py [hunk 1]
```diff
@@ -128,9 +128,9 @@ def data(self, index, role):
 
         if role == Qt.ForegroundRole:
             if item.itemData[1] == 'removed':
-                return QVariant(QColor(Qt.red))
+                return QVariant(QColor(Qt.red)) if uses_dark_mode() else QVariant(QColor(Qt.darkRed))
```
[the remark this answers is not in the document]
Comment (samuel-w) [reply src/vorta/views/diff_result.py]: It's only called in two areas,  here and in icon colourization. I did change it so `uses_dark_mode` is only called once in `diff_result`

**Comment by samuel-w** [comment]
Now the colours are object variables, and dark_mode is only called at init

**Review by m3nu — APPROVED** [review]

**Pull request merged**